src/cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/adyenGivingComponent.isml

```diff
@@ -1,10 +1,10 @@
 <isif condition="${pdict.adyen && pdict.adyen.adyenGivingAvailable}">
     <isset name="AdyenHelper" value="${require('*/cartridge/scripts/util/adyenHelper')}" scope="pdict"/>
     <link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
+    <script src="${pdict.AdyenHelper.getCheckoutUrl()}"></script>
```

Comment:
Should we use local script instead of external?

Reply:
Yes! I propose moving it to a separate PR so we can build on top of this one.